tests/data/diagnosis_summary.jsonl

```diff
@@ -1,2 +1,2 @@
+{"Category": "FailedTest", "Defective Details": "mem-bw/D2H_Mem_BW:0_miss,mem-bw/D2H_Mem_BW:1_miss,mem-bw/D2H_Mem_BW:2_miss,mem-bw/D2H_Mem_BW:3_miss,mem-bw/D2H_Mem_BW:4_miss,mem-bw/D2H_Mem_BW:5_miss,mem-bw/D2H_Mem_BW:6_miss,mem-bw/D2H_Mem_BW:7_miss,mem-bw/H2D_Mem_BW:0_miss,mem-bw/H2D_Mem_BW:1_miss,mem-bw/H2D_Mem_BW:2_miss,mem-bw/H2D_Mem_BW:3_miss,mem-bw/H2D_Mem_BW:4_miss,mem-bw/H2D_Mem_BW:5_miss,mem-bw/H2D_Mem_BW:6_miss,mem-bw/H2D_Mem_BW:7_miss,mem-bw/return_code(VAL: 1.0000 Rule:lambda x:x>0)", "kernel-launch/event_overhead:0": 0.0, "kernel-launch/event_overhead:1": -0.0016778523, "kernel-launch/event_overhead:2": -0.0654362416, "kernel-launch/event_overhead:3": -0.0771812081, "kernel-launch/event_overhead:4": -0.0067114094, "kernel-launch/event_overhead:5": -0.0117449664, "kernel-launch/event_overhead:6": -0.0402684564, "kernel-launch/event_overhead:7": -0.0100671141, "kernel-launch/return_code": 0, "kernel-launch/wall_overhead:0": 0, "kernel-launch/wall_overhead:1": 0, "kernel-launch/wall_overhead:2": 0.0194931774, "kernel-launch/wall_overhead:3": 0.022417154, "kernel-launch/wall_overhead:4": 0.0360623782, "kernel-launch/wall_overhead:5": -0.0194931774, "kernel-launch/wall_overhead:6": 0.0185185185, "kernel-launch/wall_overhead:7": 0.0438596491, "mem-bw/D2H_Mem_BW:0": "N/A", "mem-bw/D2H_Mem_BW:1": "N/A", "mem-bw/D2H_Mem_BW:2": "N/A", "mem-bw/D2H_Mem_BW:3": "N/A", "mem-bw/D2H_Mem_BW:4": "N/A", "mem-bw/D2H_Mem_BW:5": "N/A", "mem-bw/D2H_Mem_BW:6": "N/A", "mem-bw/D2H_Mem_BW:7": "N/A", "mem-bw/H2D_Mem_BW:0": "N/A", "mem-bw/H2D_Mem_BW:1": "N/A", "mem-bw/H2D_Mem_BW:2": "N/A", "mem-bw/H2D_Mem_BW:3": "N/A", "mem-bw/H2D_Mem_BW:4": "N/A", "mem-bw/H2D_Mem_BW:5": "N/A", "mem-bw/H2D_Mem_BW:6": "N/A", "mem-bw/H2D_Mem_BW:7": "N/A", "mem-bw/return_code": 1, "Index": "sb-validation-03"}
```

Comment:
why jsonl still uses "Index" instead of "index" like other formats?

Reply:
updated, thanks